.github/release-drafter.yml

```diff
@@ -0,0 +1,5 @@
+# https://github.com/jenkinsci/.github/blob/master/.github/release-drafter.adoc
+_extends: .github
+version-template: $MAJOR.$MINOR.$PATCH
+tag-template: hashicorp-vault-$NEXT_PATCH_VERSION
```

Comment:
Shouldn't this be `gitlab-branch-source-$NEXT_PATCH_VERSION`? @casz

Reply:
I thought I updated it 😭

Reply:
It's okay

Reply:
I think we can do it directly on `master`. And add the commit to this pr as comment.

Reply:
Nah PR, add skip-changelog 😅 
See config here: https://github.com/jenkinsci/.github/blob/master/.github/release-drafter.yml